### DISCUSSION TITLE
set defaultTheme by specifying theme-name

### DISCUSSION THREAD

**Discussion by silveltman** [RyanClementsHax/tailwindcss-themer#76] [Ideas]
I think it would be cool do do something like this:
```
defaultTheme: 'light',
themes: [
{
	name: 'light',
	extend: {
		colors: {
			base: getRadixColors(radixColors.gray)
		}
	}
},
{
	name: 'dark',
	extend: {
		colors: {
			base: getRadixColors(radixColors.grayDark)
		}
	}
},
```

Related to https://github.com/RyanClementsHax/tailwindcss-themer/discussions/73 , but a different approach

**Comment by RyanClementsHax** [comment]
Just to make sure I understand, you're specifying to use the `defaultTheme` value to specify which of the themes in the `themes` array should be considered default?
